src/schema/v2/quizArtworkConnection.ts

```diff
@@ -31,14 +31,19 @@ export const quizArtworkConnection: GraphQLFieldConfig<any, ResolverContext> = {
   }),
   resolve: ({ quiz_artworks }, args) => {
     const { page, size, offset } = convertConnectionArgsToGravityArgs(args)
+
     const totalCount = quiz_artworks.length
 
+    const sorted = quiz_artworks.sort((a, b) => {
+      return a.position - b.position
+    })
```

Comment:
These will just be sorted by default now. The Gravity endpoint is probably what should be doing this but 🤷